tests/test_persist_on_failure.py

```diff
@@ -62,16 +59,11 @@ def test_persists_with_overlength_field():
     # Length is max field length
     assert len(failed_task_object.exc) == 255
     # Ellipses are put in the middle
-    if six.PY2:
-        # Ellipses offset by one due to the u' in the ValueError
-        assert failed_task_object.exc.startswith("ValueError(u'")
-        assert failed_task_object.exc[124:133] == '037...590'
-    else:
-        assert failed_task_object.exc.startswith("ValueError('")
-        assert failed_task_object.exc[124:133] == '370...590'
+    assert failed_task_object.exc.startswith("ValueError('")
+    assert failed_task_object.exc[124:133] == '370...059'
```

Comment:
@morenol Then this test will need conditions for 3.5 and 3.8 to pass then. or we may have to call this locally `exc = _truncate_to_field(FailedTask, 'exc', repr(ValueError(overlong_message)))` and then do this `assert failed_task_object.exc[124:133] == exc[124:133]`